brainatlas_api/list_atlases.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+from rich.table import Table
+from rich import print as rprint
+
+from brainatlas_api import config
+from brainatlas_api.bg_atlas import (
```

Comment:
Done, I've added `__all__` to `bg_atlas` and this can be used to get all relevant classes 
```
available_atlases = [
    cls for cls in map(bg_atlas.__dict__.get, bg_atlas.__all__)
]
```